htlcswitch/link.go

```diff
@@ -1028,8 +986,8 @@ out:
 			// If the downstream packet resulted in a non-empty
 			// batch, reinstate the batch ticker so that it can be
 			// cleared.
-			if l.batchCounter > 0 && maybeBatchTick == nil {
-				maybeBatchTick = l.cfg.BatchTicker.Start()
+			if l.batchCounter > 0 && !l.cfg.BatchTicker.IsActive() {
```

Comment:
shouldn't need to check `IsActive` here?

Reply:
I don't think we want to resume if we're already active, could cause us to reinitialize a ticker every time the batch is non-empty

Reply:
We could make `Resume` a no-op if the ticker is already active.

Reply:
oh great idea!